test/WebSites/FiltersWebSite/AuthorizeBasicMiddleware.cs

```diff
@@ -4,10 +4,11 @@
 using System;
 using System.Security.Claims;
 using System.Threading.Tasks;
+using Microsoft.AspNet.Authentication;
+using Microsoft.AspNet.Authentication.Infrastructure;
```

Comment:
The middleware base class pulls that in, Infrastructure is getting nuked soon and rolled into the base namespace which will get rid of one using.